### ISSUE TITLE
[Feature Request] Download the contents of a share into a directory ignoring the share's root folder

### ISSUE THREAD

**Issue by remailednet** [tanaikech/goodls#8]
For example:
If https://drive.google.com/drive/folders/123123123 is 'Folder_1' with the contents of 'SubFolder_1', 'file_1', 'file_2'

```
goodls -u https://drive.google.com/drive/folders/123123123 -d /localfolder/here/
```

I would like the end result to be:
```
/localfolder/here/SubFolder_1
/localfolder/here/SubFolder_1/file_1
/localfolder/here/SubFolder_1/file_2
```
and not:
```
/localfolder/here/Folder_1/SubFolder_1
/localfolder/here/Folder_1/SubFolder_1/file_1
/localfolder/here/Folder_1/SubFolder_1/file_2
```

Possible to add a switch to ignore the root folder of the share? 
Or possibly if -d contains a trailing slash, ignore the root folder of the share?

**Comment by tanaikech** [comment]
Thank you for the feature request. This was implemented at v1.2.5. Please check it. https://github.com/tanaikech/goodls#v125
In your case, please test the following command.

```bash
$ goodls -u https://drive.google.com/drive/folders/123123123 -d /localfolder/here/ -ntd
```

**Comment by remailednet** [comment]
@tanaikech Awesome. I did some testing on my side and that feature works as expected for me.

Thanks!
